src/main/resources/META-INF/persistence.xml

```diff
@@ -4,7 +4,7 @@
   <persistence-unit name="pu" transaction-type="RESOURCE_LOCAL">
     <provider>org.eclipse.persistence.jpa.PersistenceProvider</provider>
     <class>entities.RenameMe</class>
-    <class>entities.Car</class>
```

Comment:
What the $%!#
Jeg har ikke rørt persistence.xml. Hvorfor sletter den Car fra den?

Reply:
havde du nogen mergekonflikt?